commcare/src/main/java/org/motechproject/commcare/service/impl/CommcareCaseServiceImpl.java

```diff
@@ -280,7 +280,7 @@ private CaseInfo populateCaseInfo(CaseJson caseResponse, String configName) {
 
         String caseType = properties.get("case_type");
         String dateOpened = properties.get("date_opened");
-        String ownerId = properties.get("owner_id)");
+        String ownerId = properties.get("owner_id");
```

Comment:
can we extend CommcareCaseServiceImplTest to test fr this field, not only for CaseId as it does currently?